config-connector/solutions/networking/helm/autoneg/README.md

```diff
@@ -0,0 +1,142 @@
+# Autoneg
+
+==================================================
+
+## NAME
+
+  autoneg
+
+## SYNOPSIS
+
+Autoneg solution uses network endpoint groups to provision load balancing across multiple clusters.
+This solution uses `./cluster/templates/autoneg.yaml` from [gke-autoneg-controller](https://github.com/GoogleCloudPlatform/gke-autoneg-controller).
+For demonstration purposes it uses a docker image with a simple Node.js service (bulankou/node-hello-world) with a single endpoint that prints out a message.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository
+  
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the autoneg folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/networking/helm/autoneg
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector
+1. [Helm](../../../README.md#helm)
```

Comment:
I'm using KCC installed in project A (a fresh installation of KCC so no other APIs are enabled) to manage GCP resources in project B (a brand new project with nothing configured), and the requirements are not clear to me that:
1. What roles does the "cnrm-system" GSA need to have in project B in order to run the solution?
2. What APIs should I manually enable in project A and/or project B?

Reply:
Thanks. I've added the note that this solution assumes that all resources are installed in the same project, where the cluster with Config Connector is installed, and that load balancing resources are installed on the same cluster where Config Connector is installed. 
It is best to simplify the solution this way, because otherwise multiple steps below become more complicated, also we'll need to test it this way too, which is extra work.